docs/src/man/reshaping_new.md

```diff
@@ -0,0 +1,337 @@
+   6 │     5  male    Age              35
+   7 │     6  male    Age              53
+   8 │     7  male    Age              35
+  ⋮  │   ⋮      ⋮            ⋮             ⋮
+  34 │     3  male    Saving accounts  little
+  35 │     4  male    Saving accounts  little
+  36 │     5  male    Saving accounts  missing
+  37 │     6  male    Saving accounts  quite rich
+  38 │     7  male    Saving accounts  little
+  39 │     8  male    Saving accounts  rich
+  40 │     9  male    Saving accounts  little
+                                   25 rows omitted
+```
+
+We can rename columns:
+
+```jldoctest dataframe
+julia> stack(german, Not([:Sex, :id]), variable_name=:x, value_name=:y)
+40×4 DataFrame
+ Row │ id     Sex     x                y
+     │ Int64  String  String           Any
+─────┼────────────────────────────────────────────
+   1 │     0  male    Age              67
+   2 │     1  female  Age              22
+   3 │     2  male    Age              49
+   4 │     3  male    Age              45
+   5 │     4  male    Age              53
+   6 │     5  male    Age              35
+   7 │     6  male    Age              53
+   8 │     7  male    Age              35
+  ⋮  │   ⋮      ⋮            ⋮             ⋮
+  34 │     3  male    Saving accounts  little
+  35 │     4  male    Saving accounts  little
+  36 │     5  male    Saving accounts  missing
+  37 │     6  male    Saving accounts  quite rich
+  38 │     7  male    Saving accounts  little
+  39 │     8  male    Saving accounts  rich
+  40 │     9  male    Saving accounts  little
+                                   25 rows omitted
+```                                           
+
+## The basics of `unstack`
+
+*Unstack* converts the data frame `german` from it long to wide format. Row and column 
+keys will be ordered in the order of their first appearance.    
+
+```jldoctest dataframe
+julia> long = stack(german, [:Age, :Sex, :Job, :Housing])
+40×4 DataFrame
+ Row │ id     Saving accounts  variable  value
+     │ Int64  String?          String    Any
+─────┼─────────────────────────────────────────
+   1 │     0  missing          Age       67
+   2 │     1  little           Age       22
+   3 │     2  little           Age       49
+   4 │     3  little           Age       45
+   5 │     4  little           Age       53
+   6 │     5  missing          Age       35
+   7 │     6  quite rich       Age       53
+   8 │     7  little           Age       35
+  ⋮  │   ⋮           ⋮            ⋮        ⋮
+  34 │     3  little           Housing   free
+  35 │     4  little           Housing   free
+  36 │     5  missing          Housing   free
+  37 │     6  quite rich       Housing   own
+  38 │     7  little           Housing   rent
+  39 │     8  rich             Housing   own
+  40 │     9  little           Housing   own
+                                25 rows omitted
+```
+
+We can even skip passing the `:variable` and `:value` values as positional arguments, as they 
```

Comment:
as commented above - better start with the example where all arguments are passed.